src/status_im2/navigation/events.cljs

```diff
@@ -140,3 +139,11 @@
                                            key-uid
                                            :keycard-pairing]))]
     {:set-root (if keycard-account? :multiaccounts-keycard :multiaccounts)}))
+
+(rf/defn dismiss-all-overlays
+  {:events [:dissmiss-all-overlays]}
+  [{:keys [db]}]
+  {:dissmiss-all-overlays-fx nil
+   :db                       (-> db
+                                 (dissoc :popover/popover)
```

Comment:
what about bottom sheet ?

Reply:
hi @flexsurfer, Thank you for the reviewing PR.

I just skipped that one, because for all overlays `(navigation/dissmiss-all-overlays)` works, except for above two. For these two we also have to remove them from db. (not sure why)

Reply:
added now